lib/active_merchant/billing/gateways/decidir.rb

```diff
@@ -107,8 +107,8 @@ def scrub(transcript)
           gsub(%r((\"card_number\\\":\\\")\d+), '\1[FILTERED]').
           gsub(%r((\"security_code\\\":\\\")\d+), '\1[FILTERED]').
           gsub(%r((\"emv_issuer_data\\\":\\\")\d+), '\1[FILTERED]').
-          gsub(%r((\"cryptogram\\\":\\\"/)\w+), '\1[FILTERED]').
-          gsub(%r((\"token_card_data\\\":{\\\"token\\\":\\\")\d+), '\1[FILTERED]')
+          gsub(%r((\"cryptogram\\\":\\\")\w+), '\1[FILTERED]').
+          gsub(%r((\"token_card_data\\\":{.*\\\"token\\\":\\\")\d+), '\1[FILTERED]')
```

Comment:
With this condition we make sure to `scrub` the `token` in any position that is within `token_card_data`

Reply:
nice